app/controllers/publishers_controller.rb

```diff
@@ -373,6 +372,11 @@ def authenticate_via_token
         flash[:alert] = t(".email_confirmed", email: publisher.email)
       end
 
+      if publisher.uphold_connection.blank?
+        publisher.uphold_connection = UpholdConnection.new(publisher: publisher)
+        publisher.save
+      end
```

Comment:
Consider using the .create(!) method

Reply:
Good idea!
https://github.com/brave-intl/publishers/pull/1825/commits/2af6ad6de7c0c3bc4e7bf2fc716a670e007a4ac0